sdk/identity/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
```

Be consistent with the rest
```suggestion
cmake_minimum_required (VERSION 3.13)
```